test/postman/dbp-test-collection.json

```diff
@@ -1705,31 +1237,25 @@
 		{
 			"id": "1450729d-5636-4a8a-b062-816497abeb60",
 			"key": "acsUrl",
-			"value": "{{dbpurl}}/alfresco",
+			"value": "alfresco-cs-repository.{{dbpurl}}/alfresco",
```

Comment:
Since `{{dbpurl}}` is no longer a URL but merely a host name, I am wondering if we should rename this parameter to make it clear this is the case.

Reply:
Can we have the protocol here instead of each of the tests? And have it as a variable so we can give it in the tests? like {{protocol}}